contracts/governance/extensions/GovernorCountingOverridable.sol

```diff
@@ -120,7 +125,13 @@ abstract contract GovernorCountingOverridable is GovernorVotes {
         return totalWeight;
     }
 
-    /// @dev Variant of {Governor-_countVote} that deals with vote overrides.
+    /**
+     * @dev Variant of {Governor-_countVote} that deals with vote overrides.
+     *
+     * NOTE: Casting a vote through {castVote} (or similar) uses the power the account has delegated to. On the
+     * opposite, casting an override with {castOverrideVote} (or similar) uses the power of the account itself as a
+     * single voter. Consequently, overriding a vote does not count as voting until the account casts a vote.
```

Comment:
I'm not sure about the "until the account casts a vote" part. It implies that the using casting a vote would sudently change the fact that the override now counts as voting, when the reality is the override never counts as voting, and the vote casting is what counts.

```suggestion
     * single voter. Consequently, overriding a vote does not count as voting.
```